kmip/tests/unit/core/primitives/test_date_time.py

```diff
@@ -91,7 +91,7 @@ def test_str(self):
         properly.
         """
         t = (2015, 8, 11, 9, 18, 55, 1, 223, 1)
-        s = "Tue Aug 11 09:18:55 2015"
+        s = str(time.ctime(int(time.mktime(t))))
```

Comment:
This change effectively inlines the code used by DateTime to convert to a string (see [DateTime:\__str__](https://github.com/OpenKMIP/PyKMIP/blob/20dbad5/kmip/core/primitives.py#L955)). I'm not convinced that this test case is useful in this case.